src/ServiceControl.Audit.Persistence.RavenDB/Indexes/MessagesViewIndex.cs

```diff
@@ -12,23 +12,24 @@ public MessagesViewIndex()
         {
             Map = messages =>
                 from message in messages
+                let metadata = message.MessageMetadata
```

Comment:
@mauroservienti I was banging my head against the wall... but I think let somehow is not supported. Could you quickly review the CI build? The error is about object but it does not make sense to me.

Issue it to that I for some reason cannot run these test locally so I cannot easily run them